doc/_static/switcher.json

```diff
@@ -0,0 +1,105 @@
+[
+    {
+        "name": "stable/3.5.1",
+        "version": "stable"
+    },
+    {
+        "name": "devel/3.6",
```

Comment:
```suggestion
        "name": "3.6 (devel)",
```

Reply:
.. and I changed this to `"3.6 (devdocs)"` to be consistent with my argument above ;-)  However, I don't feel strongly if we should rename these.  

I _do_ hope we can merge this, just to see if it works on the devdocs.  We can always fine-tune it later.